changelog/unreleased/other/6928.md

```diff
@@ -0,0 +1 @@
+- :warning: Knows issue: the sdk might crash when no locations are available. The fix will be in the upcoming releases.
```

Comment:
Is there a workaround we could do?

Reply:
I'm waiting for the reply from Ramon about it. But not sure: the crash produces under the hood in the common sdk.